### DISCUSSION TITLE
镜像仓库是http协议的，overlaybd快照得在哪里设置成http

### DISCUSSION THREAD

**Discussion by jinteng123** [containerd/overlaybd#281] [General]
![image](https://github.com/containerd/overlaybd/assets/22409094/a81acc6f-5961-4788-9932-94aa077eace8)
failed to open remote file https://registry.common.svc.qzprod:8092/v2/tomcat/blobs/sha256:39b9b405c53f706ab111cd8f05ee01aa1ef610118f2d7e48659934cb56ccad22: unknown

**Comment by WaberZhuang** [comment]
HTTP is not currently supported, but we may provide it if you strongly desire, as a global configuration option or `rpull` command option.

**Reply by jinteng123** [reply]
需要这个配置，内部很多仓库都是http的
